test/spec/modules/fledgeForGpt_spec.js

```diff
@@ -97,17 +113,90 @@ describe('fledgeForGpt module', () => {
     it('sets GPT slot config for each ad unit that has PAAPI config', () => {
       const cfg = {
         au1: {
-          componentAuctions: ['a1', 'a2']
+          componentAuctions: [{seller: 's1'}, {seller: 's2'}]
         },
         au2: {
-          componentAuctions: ['a3']
+          componentAuctions: [{seller: 's3'}]
         }
       }
       getPAAPIConfig.returns(cfg);
       setPAAPIConfigForGPT('mock-filters');
       sinon.assert.calledWith(getPAAPIConfig, 'mock-filters');
       Object.entries(cfg).forEach(([au, config]) => {
-        sinon.assert.calledWith(setGptConfig, au, config.componentAuctions);
+        sinon.assert.calledWith(setGptConfig, au, configsBySeller(config.componentAuctions));
+      })
+    });
+
+    describe('when reuse = false', () => {
+      it('resets all previously set configs', () => {
+        getPAAPIConfig.returns({
+          au1: {
+            componentAuctions: [{seller: 's1'}, {seller: 's2'}]
+          },
+          au2: {
+            componentAuctions: [{seller: 's1'}]
```

Comment:
If `s2` is added here the test would fail. I think setGptConfig for au2 will reset both, but we actually want the new `s2`